boilerplate/openshift/golang-osd-operator/README.md

```diff
@@ -1,5 +1,14 @@
 # Conventions for OSD operators written in Go
```

Comment:
Can we add another commit to this linking this documentation from our main README?  I didn't even know this was here until reviewing this PR.

Reply:
Yeah, good idea.

Reply:
Done.